pkg/agent/tunnelserver/tunnelserver.go

```diff
@@ -216,6 +217,26 @@ func (t *tunnelServer) GitCredentials(ctx context.Context, message *tunnel.Messa
 	return &tunnel.Message{Message: string(out)}, nil
 }
 
+func (t *tunnelServer) GitSSHSignature(ctx context.Context, message *tunnel.Message) (*tunnel.Message, error) {
```

Comment:
You'll also need to implement this function in ./proxyserver (just forward the message) for Pro setups